examples/async.rs

```diff
@@ -1,43 +0,0 @@
-#![no_std]
```

Comment:
Does this mean that this pull request prevents implementing this example? I found it great to show how to perform unrelated tasks in parallel.

Reply:
No, I just didn't want to increase the number of examples demonstrating the same thing.

Reply:
I don't think the new `timer_parallel` example demonstrates the same thing though. It doesn't show the interaction with buttons.

Reply:
We plan to add a small "game" with LEDs and buttons in the future.